doc/USER_GUIDE.md

```diff
@@ -296,6 +296,57 @@ $ curl -sSL "http://localhost:8001/api/v1/nodes/ip-192-168-92-220.us-east-2.comp
 
 ---
 
+## Customizing Containerd Config
+
+In some cases, customers may want to customize the [containerd configuration](https://github.com/containerd/containerd/blob/main/docs/man/containerd-config.toml.5.md)
+on their nodes. We utilize the drop-in style `import` files [supported by containerd](https://github.com/containerd/containerd/blob/main/docs/man/containerd-config.toml.5.md#format)
+to achieve this. 
+
+**Import a containerd config**
+
+If you have a containerd config of your own, you can place it inside the directory `/etc/containerd/config.d/` via your user data.
+Make sure that you explicitly set the `.toml` extension on the file.
+
+> [!NOTE]
+> When adding a new `containerd` config file to the `import` path after `containerd` has already started (i.e. `bootstrap.sh` already ran),
+> you'll need to restart `containerd` to pick up the latest configuration.
+
+The `bootstrap.sh` script will write the defaults for the containerd config to `/etc/containerd/config.toml`, containing an `import` 
+list containing the path glob: `/etc/containerd/config.d/*.toml`. `bootstrap.sh` will also create this directory,
+so the user is not required create it when executing commands after the bootstrap is run.
```

Comment:
Don't think we need to explain all this here, but lets make sure that directory is created when the AMI is built, not by the bootstrap script.
```suggestion
```

Reply:
i don't want to omit any info about the pattern/file-ext, Ill just change the pattern to `/etc/containerd/config.d/*` and then ill make this change

Reply:
Sure, I don't feel too strongly about that detail, but I don't think anyone would expect files without a `.toml` extension to be included, all containerd config files are TOML files.